### ISSUE TITLE
With the latest version of NVDA (2025.2), OSARA no longer works at all.

### ISSUE THREAD

**Issue by Ambro86** [jcsteh/osara#1299]
Hello, I have just updated NVDA to version 2025.2. Osara no longer works in any way; it doesn’t read anything at all. I also tried reinstalling it, but nothing has changed. Is there any way to solve this problem? Thanks.

**Comment by ScottChesworth** [comment]
Check whether you have the LBL add-on. If you do, either update it (being sure to read their usage terms) or uninstall if you don't use it.
Let us know how that goes.

**Comment by Ambro86** [comment]
I hadn’t installed the lbl addon, I tried installing the updated version
but I noticed that nothing changed. At that point, I uninstalled them. I
should note that with Jaws Osara it works fine, but with NVDA it doesn’t
read anything.

Il giorno dom 17 ago 2025 alle ore 22:39 ScottChesworth <
***@***.***> ha scritto:

**Comment by jcsteh** [comment]
Do you have any other NVDA add-ons installed? If so, could you please try restarting NVDA with all add-ons disabled and see if that resolves the problem?

I can't reproduce this with NVDA 2025.2, hence my questions.

**Comment by ScottChesworth** [comment]
Please let us know which add-on the conflict is with when you figure that out.
